# message with IMU data from ICM20948
#
# Calculated Angles
float32 yaw
float32 pitch
float32 roll
#
#Accelorometer
int16 acc_x
int16 acc_y
int16 acc_z
#
#Gyroscope
int16 gyro_x
int16 gyro_y
int16 gyro_z
#
#Magnetometer
int16 mag_x
int16 mag_y
int16 mag_z